docs/hub/datasets-libraries.md

```diff
@@ -0,0 +1,16 @@
+# Libraries
+
+The Dataset Hub has support for several libraries in the Open Source ecosystem.
+Thanks to the `huggingface_hub` Python library, it's easy to enable sharing your datasets on the Hub.
+The Hub supports many libraries, and we're working on expanding this support!
+We're happy to welcome to the Hub a set of Open Source libraries that are pushing Machine Learning forward.
+
+The table below summarizes the supported libraries and their level of integration.
+
+| Library                                                                     | Description                                                                                                         | Download from Hub | Push to Hub |
+|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|---|----|
```

Comment:
should we also link to the guides you wrote?

Reply:
You have the links on the left tab when you're on this page